incubator-tools/overlapping_split/overlapping_split.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #255.                                    document.entities.append(new_entity)
</pre><p>This whole section is very indented/difficult to read. Please refactor.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Refactored the code.